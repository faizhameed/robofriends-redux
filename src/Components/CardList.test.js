import { shallow } from "enzyme";
import React from "react";
import CardList from "./CardList";
import toJson from "enzyme-to-json";

it("expect to run card componen", () => {
  const mockRobots = [
    {
      id: 1,
      name: "John Snow",
      username: "john",
      email: "joh@gmail.com",
    },
  ];
  const wrapper = shallow(<CardList robots={mockRobots} />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
