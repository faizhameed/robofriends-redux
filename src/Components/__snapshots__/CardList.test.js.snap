// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to run card componen 1`] = `
<div>
  <Card
    email="joh@gmail.com"
    id={1}
    key="0"
    name="John Snow"
    username="john"
  />
</div>
`;
